Marlin/src/lcd/language/language_de.h

```diff
@@ -394,6 +409,7 @@
   #define MSG_FILAMENT_CHANGE_INIT_1          _UxGT("Bitte warten...")
   #define MSG_FILAMENT_CHANGE_UNLOAD_1        _UxGT("Auswerfen...")
   #define MSG_FILAMENT_CHANGE_INSERT_1        _UxGT("Laden und Klick")
+  #define MSG_FILAMENT_CHANGE_HEAT_1          _UxGT("Klicke zum Heizen")
```

Comment:
Eearlier you used "Klick" and here you use "Klicke". Both forms are correct, but you should stay by one.

Reply:
is the context when using "Klick" or "Klicke"

Reply:
is the context when using "Klick" or "Klicke"